mappings/net/minecraft/entity/ai/AiGoalTrackTarget.mapping

```diff
@@ -0,0 +1,32 @@
+CLASS xu net/minecraft/entity/ai/AiGoalTrackTarget
+	FIELD a checkCanNavigate Z
+	FIELD b canNavigateFlag I
```

Comment:
It is a value that can be 0, 1, or 2.  It's a strange way to do things, so naming it isn't terribly straightforward. I think `flag` is the right term for this kind of value.